### PR TITLE
Create installation / setup page

### PR DESCRIPTION
**Pull request by arjunsavel**
- Install the package
- set up the ADS token (addresses #31)

I want to check that I have all dependencies accounted for this way? Will compare against the github workflows.

### CODE REVIEW

**Comment by arjunsavel** [comment]
Docs built here: https://astrodb-scripts.readthedocs.io/en/more_docs/index.html

note that the version tag is super long in the top left. that'll be fixed on merge.

File: docs/pages/installation.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+.. code-block:: bash
+
+    pip install astrodb_utils
+
+Installing from source
+-----------------------
+
+We develop `astrodb_utils` on `GitHub <https://github.com/astrodbtoolkit/astrodb_utils>`_.
+If you received the code as a tarball or zip, feel free to skip the first two lines; they essentially download the source code.
```
Comment (dr-rodriguez) [inline docs/pages/installation.rst:16]: Technically, it's the third line that downloads the source code.

File: docs/pages/installation.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
+Installation and setup
+===========================
+
+Installing with pip
+-----------------------
+`astrodb_utils` is distributed on `PyPI <https://pypi.org/>`_. It can be installed with
```
Comment (dr-rodriguez) [inline docs/pages/installation.rst:6]: Would a link to astrodb_utils in PyPI be better than just the homepage?

File: docs/pages/installation.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    python3 -m pip install -U pip
+    python3 -m pip install -U setuptools setuptools_scm pep517
+    git clone https://github.com/astrodbtoolkit/astrodb_utils.git
+    cd astrodb_utils
+    python3 -m pip install -e .
+
+
+Set up ADS token
+-----------------------
+`astrodb_utils` can query `NASA Astrophysics Data System <https://ui.adsabs.harvard.edu/>`_ with the `ingest_publications` function.
```
Comment (dr-rodriguez) [inline docs/pages/installation.rst:31]: Missing `the` in "... can query [the] NASA ..."

File: docs/pages/installation.rst [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+    pip install -e ".[test]"
+
+Then, in the tests directory, run
+
+.. code-block:: bash
+
+    git clone git@github.com:astrodbtoolkit/astrodb-template-db.git
+
+This step installs a template repository. Tests can then be run in the top-level directory.
```
Comment (dr-rodriguez) [inline docs/pages/installation.rst:61]: Should add explicit instructions on what to run (ie, pytest) and where to actually run the tests.

File: docs/pages/organization.rst [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Repository / package organization
```
Comment (dr-rodriguez) [inline docs/pages/organization.rst:1]: I think a link is missing somewhere- I can't find this page in the generated docs

File: docs/pages/organization.rst [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+====================================
+The below schematic image shows the organization of the repository and the packages.
+
+![Repo organization](repo_organization.png)
```
Comment (dr-rodriguez) [inline docs/pages/organization.rst:5]: I thought this figure was for our internal notes/organization, not really something to show to the public. The gist of it is good, but I think we can create a better/clearer image.

Comment (arjunsavel) [reply docs/pages/organization.rst:5]: Ah, good point! I'll think about how to mock something up.

Comment (dr-rodriguez) [reply docs/pages/organization.rst:5]: We can probably merge-as-is and then revisit this figure.

**Review by dr-rodriguez — APPROVED** [review]

**Pull request merged**